### ISSUE TITLE
./build.sh all hangs during installation of npm packages

### ISSUE THREAD

**Issue by theScrabi** [OpenSlides/OpenSlides#5643]
**Describe the bug**
Command `./build.sh all` hangs while running the command `ng build --prod "--output-path" "/build/app/static"`. After some timeout ng is killed and the installation process failes. I'm running the setup on a Hetzner CX11 instance. Could it be that it does not have the power to run it correclty?

**How to Reproduce**
1. Get a Hetzner CX11 instance
2. Install official docker
2. `git clone https://github.com/OpenSlides/OpenSlides.git`
3. run `./build.sh all`

**Expected behavior**
Compilation runs through so one can run `docker-compose up`

**System information**

- OpenSlides-Version: [current master]
- Additional version information (if any): [e.g. commit hash or installation-month]
- Python Version: [none]
- Device: [Hetzner CX11]
- OS: [ubuntu 20.04.1]

**Screenshots**
```
...

Step 15/19 : RUN npm run build-to-dir /build/app/static
 ---> Running in 70536dd9fb5b

> OpenSlides3-Client@3.2.0 build-to-dir /build/app
> npm run build -- --output-path "/build/app/static"


> OpenSlides3-Client@3.2.0 build /build/app
> ng build --prod "--output-path" "/build/app/static"

Warning: Entry point 'ngx-papaparse' contains deep imports into '/build/app/node_modules/papaparse/papaparse.min.js'. This is probably not a problem, but may cause the compilation of entry points to be out of order.
Generating ES5 bundles for differential loading...
Killed
npm ERR! code ELIFECYCLE
npm ERR! errno 137
npm ERR! OpenSlides3-Client@3.2.0 build: `ng build --prod "--output-path" "/build/app/static"`
npm ERR! Exit status 137
npm ERR! 
npm ERR! Failed at the OpenSlides3-Client@3.2.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/openslides/.npm/_logs/2020-10-19T16_21_03_004Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 137
npm ERR! OpenSlides3-Client@3.2.0 build-to-dir: `npm run build -- --output-path "/build/app/static"`
npm ERR! Exit status 137
npm ERR! 
npm ERR! Failed at the OpenSlides3-Client@3.2.0 build-to-dir script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/openslides/.npm/_logs/2020-10-19T16_21_03_154Z-debug.lo
```

**Comment by tsiegleauq** [comment]
> Could it be that it does not have the power to run it correclty?

yes. [We used to have a flag](https://github.com/OpenSlides/OpenSlides/commit/e1acf6e9d6f23a90bd53f1b0f1544dfeb84f5d81#diff-1846122c2c83a486a3693f7966aa522c34cf489f674185c4da0d9221683fd81fL13) for increasing the minimum amount of necessary RAM for `ng build --prod` to at least 4GB (of effectively usable memory), but I disabled it with angular 9 since ram management in npm got slightly better. Even on Linux 2GB of memory is not enough to build angular applications in prod mode.
